### ISSUE TITLE
Tracker Type not correct

### ISSUE THREAD

**Issue by matsbrucke** [wolfgangimig/itol#2]
Hi,

We use different trackers in our redmine (Aufgabe, Fehler, Feature=Task, Bug, Feature). The plugin does not pull the right names for the trackers from redmine. It shows "bug, feature, support"

best

Mats

**Comment by wolfgangimig** [comment]
Yes Mats,
the addin should read the correct tracker IDs and names.

As a workaround, you can edit the IssueServiceImpl.js file in the installation directory, subdirectory app.

Enter the tracker IDs and names at this lines (hope, you know a little JavaScript):

```
var propIssueType = propertyClasses.get(Property.ISSUE_TYPE);
propIssueType.setSelectList([ new IdName(1, "Bug"),
        new IdName(2, "Feature Request"), new IdName(3, "Support") ]);
```

Regards
Wolfgang

**Comment by wolfgangimig** [comment]
Solved in version 1.1
